handlers/salesforce-disaster-recovery/src/handlers/updateZuoraAccounts.ts

```diff
@@ -3,29 +3,27 @@ import { ZuoraClient } from '@modules/zuora/zuoraClient';
 import { type AccountRow, batchUpdateZuoraAccounts } from '../services';
 
 export const handler = async (event: { Items: AccountRow[] }) => {
+	const { Items } = event;
+
 	const stage = process.env.STAGE;
 
 	if (!stage) {
 		throw new Error('Environment variables not set');
 	}
```

Comment:
```suggestion
const stage = checkDefined(
		process.env.STAGE,
		'Environment variables not set',
	);
```
You could use the same pattern here.

Reply:
Ah nice utility function. I have updated the code.